### DISCUSSION TITLE
Gatsby not working well when disabled javascript

### DISCUSSION THREAD

**Discussion by JogiPrasadPakki** [gatsbyjs/gatsby#30866] [Help] [answered]
## Description

I am trying find seo errors on my website  by disableing java script how actual bots see my site this shows wired result.

### Steps to reproduce
I used common layout file to provide navbar and fotter to all my pages as like 
```javascript
export default function Layout(props) {
  return(
    <>
        <NavBar/>
         {props.children}
         <FotterBar/>
    </>
  )
}
```
and it linked to **gatsby-browser.js**
```javascript
export const wrapPageElement = ({ element, props }) => {
  return <Layout {...props}>{element}</Layout>;
};

```

### Expected result
I want to render both  navbar bar and fotter bar along with content when disable java script on browser 

### Actual result 
But it only render ``` {props.children}```  without navbar bar and fotter and even bootstrap cdn placed inside **react-helmet** not loading
What happened.

**Comment by LekoArts** [comment, marked as answer]
https://www.gatsbyjs.com/docs/reference/config-files/gatsby-browser/#wrapPageElement

> Note: There is an equivalent hook in Gatsby’s SSR API. It is recommended to use both APIs together. For example usage, check out Using i18n.

Adding the same code from `gatsby-browser.js` to `gatsby-ssr.js` should fix your issue
